src/utility.js

```diff
@@ -200,3 +204,54 @@ function countFlags(value) {
 	}
 	return count;
 }
+
+/**
+ * Compute compensating position offsets if body or html element has non-static position attribute.
+ * @private
+ * @param {number} windowWidth Window width in pixels.
+ * @param {number} windowHeight Window height in pixels.
+ * @return {Offsets} The top, left, right, bottom offset in pixels
```

Comment:
"Offsets" isn't a type in this context. This should be "Object". Example:

`@return {Object} An object with top, left, right, bottom offset values`